docs/eventing/samples/kafka/channel/README.md

```diff
@@ -1,156 +1,190 @@
-
-1. To setup a broker that will use the new default Kafka channels, you must
-   create a new _default_ broker, using the command:
-   ```bash
-   kubectl create -f - <<EOF
-   apiVersion: eventing.knative.dev/v1
-   kind: Broker
-   metadata:
-    name: default
-   EOF
-   ```
+## Creating a Kafka Channel
+
+- Create a Kafka Channel that contains the following YAML:
+
+    ```yaml
+    apiVersion: messaging.knative.dev/v1beta1
+    kind: KafkaChannel
+    metadata:
+      name: my-kafka-channel
+      spec:
+        numPartitions: 3
+        replicationFactor: 1
+    ```
+
+## Specifying Kafka as the default Channel implementation
+<!--TODO: Move to admin guide-->
+
+1. To configure Kafka Channel as the [default channel configuration](../../../channels/channel-types-defaults), modify the `default-ch-webhook` ConfigMap so that it contains the following YAML:
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: default-ch-webhook
+      namespace: knative-eventing
+    data:
+      # Configuration for defaulting channels that do not specify CRD implementations.
+      default-ch-config: |
+        clusterDefault:
+          apiVersion: messaging.knative.dev/v1beta1
+          kind: KafkaChannel
+          spec:
+            numPartitions: 3
+            replicationFactor: 1
+    ```
+
+1. If Kafka is configured as the default Channel implementation for your cluster, creating a Channel with the following default YAML creates a Kafka Channel:
+
+    ```yaml
+    apiVersion: messaging.knative.dev/v1
+    kind: Channel
+    metadata:
+      name: testchannel-one
+    ```
+
+1. Verify that the Channel was created properly by checking that your Kafka cluster has a `testchannel-one` Topic. If you are using Strimzi, you can run the command:
+
+    ```bash
+    kubectl -n kafka exec -it my-cluster-kafka-0 -- bin/kafka-topics.sh --bootstrap-server my-cluster-kafka-bootstrap:9092 --list
+    ```
+
+    The output looks similar to the following:
+
+    ```
+    ...
+    __consumer_offsets
+    knative-messaging-kafka.default.my-kafka-channel
+    knative-messaging-kafka.default.testchannel-one
+    ...
+    ```
+
+    The Kafka Topic that is created by the Channel contains the name of the namespace, `default` in this example, followed by the name of the Channel. In the consolidated Channel implementation, it is also prefixed with `knative-messaging-kafka` to indicate that it is a Kafka Channel from Knative.
+
+    !!! note
+        The topic of a Kafka Channel is an implementation detail and records from it should not be consumed from different applications.
+
+## Configuring the Knative Broker for use with Apache Kafka Channels
+
+The following example uses a ApiServerSource to publish events to the Broker, and a Trigger that routes events to a Knative Service.
+<!--TODO: Not sure this example makes sense, why would you have an event source AND channels?-->
+
+1. Create a Broker that uses Kafka Channels by default:
```

Comment:
There is some missing config as the broker was using `InMmemoryChannel` and it had an annotation of `MTChannelBasedBroker`

See `Set as default broker implementation` > https://knative.dev/v0.23-docs/eventing/broker/kafka-broker/

Reply:
See above comment - idk if we should even keep this and try to fix it or remove for now

Reply:
When I was trying this sample a week ago, I actually liked that all of these pieces are in one place, so that I can just follow the steps and see the complete picture without switching in different tabs and getting confused. Even though its not kafka related as you mentioned in your comment above, I think its better to justify with examples here that kafka is actually being used to do _these_ things - just my opinion. 

If the broker example stays, I think this is the configmap change that has to go in as @xtreme-sameer-vohra mentioned, because otherwise it had InMemoryChannel in there which would essentially create imc for all the brokers. But now since we have already changed the default Channel to be a Kafka channel in earlier steps, we can just simply put `Kind: Channel`
```
kubectl apply -f - <<EOF
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: config-br-default-channel
  namespace: knative-eventing
data:
  channelTemplateSpec: |
    apiVersion: messaging.knative.dev/v1
    kind: Channel
EOF
```

And related to this, I think you removed both the steps - `This will give you two pods, such as:` and `Inside the Apache Kafka cluster you should see two new topics, such as:`. Here I think deleting the pods part makes sense, but about topic creation, it should stay but it should be changed to:
```
Inside the Apache Kafka cluster you should see one new topic, such as:
knative-messaging-kafka.default.default-kne-trigger
```

Reply:
I've decided to remove all of the broker stuff from this section, period.
It's too far out of scope to start adding even more ConfigMap details here, particularly when the topic is Kafka Channels, not configuring Brokers.
The plan is to move all of these types of topics to the admin guide anyway, e.g. https://knative.dev/docs/admin/eventing/broker-configuration/, so we'd be duplicating information here to have to clean it up again later.

This was supposed to be a quick PR to do some cleanup, not an entire rewrite of this sample, so I think we should try to get the changes merged so that we have some improvement for now, and we can follow it up with more PRs later to flesh out the full use cases and which other topics need to be included or referenced.